compiler/AST/build.cpp

```diff
@@ -2310,8 +2310,8 @@ buildFunctionDecl(FnSymbol*   fn,
 
   if (optWhere)
   {
-    if (fn->hasFlag(FLAG_EXTERN))
-      USR_FATAL_CONT(fn, "Extern functions cannot have where clauses.");
+    if (fn->hasFlag(FLAG_EXPORT))
+      USR_FATAL_CONT(fn, "Export functions cannot have where clauses.");
```

Comment:
"Export" -> "Exported"?  I tend to view "export proc" as a command while "extern proc" is a descriptor, so using "export" as the adjective description of a function seems wrong to me.